src/Components/DraftFeed/Components/DraftablePlayers/DraftablePlayers.js

```diff
@@ -0,0 +1,31 @@
+import './DraftablePlayers.css';
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+export var players = [];
```

Comment:
Having a variable outside of a component like this can cause problems for react, including memory leaks. We should move this to the containing component (`DraftFeed` I believe) and then have this component take in an event handler that it uses to update the list.

Let me know if that makes sense. If not, I'd be happy to find time to hop on a video call and do some pair programming on this.